src/lbp.jl

```diff
@@ -0,0 +1,92 @@
+function lbp_original(bit_pattern::Array{Bool, 1})
+	sum([b * 2 ^ (i - 1) for (i, b) in enumerate(bit_pattern)])
+end
+
+UNIFORM_LBP_TABLE = Dict{Array{Bool, 1}, Int}()
```

Comment:
If you can, use `const` on these. Will make performance much better (type inference will work for `const` globals but not for non-const globals).